stumpwm.texi.in

```diff
@@ -2820,7 +2820,8 @@ Abort the menu.
 
 In addition, you can customize the number of items shown at a time (a
 page) with the @var{*menu-maximum-height*} variable. The default
-value, @code{nil}, means that there is no limit to the page size.
+value, @code{50}, limits the menu size to 50 items to avoid crashing
+the X server when a menu has a large amount of entries.
```

Comment:
The documentation should still explain the effect of this variable being nil (no limit to the menu page size, which can cause the x11 server to crash).

Reply:
Got it, just added a commit to document this.